### DISCUSSION TITLE
有无考虑过支持sqlite3呢

### DISCUSSION THREAD

**Discussion by zhufeng** [MoeNetwork/Tieba-Cloud-Sign#263] [想法]
目前的最新版应该只支持mysql，从软件的功能性来看，sqlite3应该也挺合适这个场景的，有无考虑过支持sqlite3呢。

**Comment by n0099** [comment]
https://z.n0099.net/#narrow/near/103557 https://github.com/n0099/open-tbm/issues/48
kenvix此前在mysqli->pdo上做出的重要指示精神选集： https://github.com/MoeNetwork/Tieba-Cloud-Sign/pull/253#issuecomment-1805933536

**Comment by BANKA2017** [comment]
先回答问题：**无**

SQLite 跟目前的 MySQL 语法上有差异，目前程序和插件内有大量硬编码的查询语句难以迁移。

此外支持 SQLite 需要通过配置对数据库文件做额外的保护，否则就是拱手将数据库送给任何访问的人，目前本人能力有限做不到也不想对任何的配置失误负责。
